AWSCognitoIdentityProvider/AWSCognitoIdentityUser.m

```diff
@@ -1743,7 +1748,7 @@ -(instancetype) initWithToken:(NSString *)token {
 }
 
 -(NSDictionary<NSString *, id> *) tokenClaims {
-    NSDictionary * result = nil;
+    NSDictionary * result = [NSDictionary dictionary];
```

Comment:
nit: it is always nice to use literal syntax for arrays and dictionaries. In this case I used this code.

`NSDictionary * result = @{};`

Reply:
done.